superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -300,8 +301,8 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
       setDB({
         type: ActionType.dbSelected,
         payload: {
-          parameters: {},
-          configuration_method: CONFIGURATION_METHOD.SQLALCHEMY_URI,
+          configuration_method: CONFIGURATION_METHOD.DYNAMIC_FORM,
```

Comment:
These are temporary, but necessary for creating an ephemeral env, will be changed after testing!